src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -280,6 +320,22 @@ public AzureAppConfigurationOptions ConfigureClientOptions(Action<ConfigurationC
             return this;
         }
 
+        /// <summary>
+        /// Configure the time to wait for a response from one replica before initiating a request to the next available replica.
+        /// </summary>
+        /// <param name="parallelRetryInterval">The time to wait for a response from one replica before trying the next available replica.</param>
+        /// <remarks>Minimum value for <paramref name="parallelRetryInterval"/> is 1 second.</remarks>
+        public AzureAppConfigurationOptions ConfigureParallelRetryTimeout(TimeSpan parallelRetryInterval)
```

Comment:
I'd like to hear everyone's thoughts on this. Do we like to make this a `ConfigureFailOver(failOverOptions)`, so it's more extensible in the future.

Reply:
Cool. Thanks, Jimmy.

Unlike refresh interval, fail-over is something people expect it just works. I'm not sure how I, as a user, would want to customize this parallel retry even if I'm given the option. I am wondering if we should leave this to the future when we see a clearer need. For now, we need to get it right out of the box and shouldn't count on a user.

Reply:
I'm cool with that as well.

Reply:
It will have default values and the user can expect it to just work.
This option will be for people who want fine grained control over their failover policies. IMO we can keep `ConfigureFailOver` option.

Reply:
What would the documentation be that we would write to discuss this property? Why would someone want to have `fine grained control`? I can't even think of a reason.

Maybe in the future we would have a better reason. It's not needed for the first release, I suggest we give ourselves some more time to think about that.

Reply:
That makes sense! Let's just use `ParallelFailOverInterval`.